src/main/scala/org/apache/mesos/chronos/scheduler/config/SchedulerConfiguration.scala

```diff
@@ -108,6 +108,17 @@ trait SchedulerConfiguration extends ScallopConf {
     descr = "Reconciliation interval in seconds",
     default = Some(600))
 
+  lazy val mesosAuthenticationPrincipal = opt[String]("mesos_authentication_principal",
+    descr = "Mesos Authentication Principal",
+    noshort = true
+  )
```

Comment:
Style consistency: Other parameters above are not separated by newlines, and the closing ')' parens is on the previous line.